From aaae611819c6a6f92a55d5c82efb8738ec9a23df Mon Sep 17 00:00:00 2001
From: Jean-Yves Avenard <jyavenard@mythtv.org>
Date: Mon, 14 Jul 2014 02:50:37 +1000
Subject: [PATCH 2/2] metadata: do not use script name for artwork file name

Clear inetref from the script name.

(cherry picked from commit ee7470d8bc0b843b80d13f4d9d92a6c3b06c3b2b)
---
 mythtv/libs/libmythmetadata/metadataimagedownload.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mythtv/libs/libmythmetadata/metadataimagedownload.cpp b/mythtv/libs/libmythmetadata/metadataimagedownload.cpp
index fb31810..eef4f2f 100644
--- a/mythtv/libs/libmythmetadata/metadataimagedownload.cpp
+++ b/mythtv/libs/libmythmetadata/metadataimagedownload.cpp
@@ -403,7 +403,7 @@ QString getDownloadFilename(VideoArtworkType type, MetadataLookup *lookup,
     }
     else if (lookup->GetType() == kMetadataVideo ||
              lookup->GetType() == kMetadataRecording)
-        title = lookup->GetInetref();
+        title = MetaGrabberScript::CleanedInetref(lookup->GetInetref());
     else if (lookup->GetType() == kMetadataGame)
         title = QString("%1 (%2)").arg(lookup->GetTitle())
                     .arg(lookup->GetSystem());
-- 
1.7.10.2

